src/components/Steps.js

```diff
@@ -1,81 +1,49 @@
 // @flow
 import React from 'react';
 import styled from 'styled-components';
-import { FormattedMessage } from 'react-intl';
+import { injectIntl, intlShape } from 'react-intl';
 import { Col, Row, Container } from 'reactstrap';
+import Step from './Step';
 
-const Steps = styled.div`
+const StepIndicatorSection = styled.div`
   background-color: ${props => props.theme.colors.helFog};
   padding-top: 4em;
+  padding-bottom: 2em;
 `;
 
 const StepContainer = styled(Container)`
-  width: 37.5em;
+  @media only screen and (min-width: 500px) {
```

Comment:
That's super dangerous, please fix these properly :) There's a good documentation how to handle media templates with Styled-components:
https://www.styled-components.com/docs/advanced#media-templates